release-tools/README.md

```diff
@@ -141,11 +141,10 @@ typically want to sanity check this with:
 
         git push -n
 
-Push new tags to public repo. Again sanity check with:
+Push new tags to public repo. You must specify the remote and tag name
+explicitly. Again sanity check with:
 
-        git push --tags -n
-
-to make sure no local tags were pushed.
+        git push -n <remote> <tagname>
```

Comment:
Following git parlance, that should really be this:

    git push -n <repository> <tagname>

Reply:
Done